com.unity.render-pipelines.high-definition/Runtime/Lighting/Light/HDAdditionalLightData.cs

```diff
@@ -2615,8 +2710,8 @@ internal void UpdateAreaLightEmissiveMesh(bool fromTimeLine = false)
             emissiveMeshRenderer.sharedMaterial.SetColor("_EmissiveColor", value);
 
             // Set the cookie (if there is one) and raise or remove the shader feature
-            emissiveMeshRenderer.sharedMaterial.SetTexture("_EmissiveColorMap", areaLightCookie);
-            CoreUtils.SetKeyword(emissiveMeshRenderer.sharedMaterial, "_EMISSIVE_COLOR_MAP", areaLightCookie != null);
+            emissiveMeshRenderer.sharedMaterial.SetTexture("_EmissiveColorMap", m_AreaLightEmissive);
```

Comment:
I am not sure to understand what is happening here: why areaLightCookie is replace by m_AreaLightEmissive ?

Reply:
if we plug only a cookie m_AreaLightEmissive == areaLightCookie.
If we plug only a cookie m_AreaLightEmissive == IESpot.
If we plug a cookie & an IES in this case m_AreaLightEmissive is a generated texture which is a product (linear blend) of IES & Cookie

Reply:
m_AreaLightEmissive contains the mips computed too.
cf. In LightCookieManager.cs
`FetchAreaCookie(CommandBuffer cmd, Texture cookie, ref Texture emissive)`
vs
`FetchAreaCookie(CommandBuffer cmd, Texture cookie, Texture ies, ref Texture emissive)`